app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  def post; end
+
+  def posts; end
```

Comment:
- [ ] you should create a separate controller for `Posts` as in the requirements and you should add `actions` like `index` and `show` in both controllers `users` and `posts`